storybook/manager-head.html

```diff
@@ -16,7 +16,7 @@
     background-color: white;
   }
 
-  data-title["Legacy"] {
-    background-color: red;
+  #kaio, [data-parent-id="kaio"], [data-id^="kaio"]{
+    display: none !important;
   }
```

Comment:
This hides the docs from storybook, which is kind of a double edge sword since we can't see it locally anymore either. So my recommendation is to only move the docs to `KAIO/` when everything is polished and done.

Reply:
Sorry, little confused with this PR now? Was this going to be merged into the rollup branch for all migrations? I think because this was rebased off that it has component updates not related to the notifications (ie: this update, brandmoment the Error page)